### ISSUE TITLE
Request construction makes more sense under section 8

### ISSUE THREAD

**Issue by shobitb** [decentralized-identity/decentralized-web-node#162]
The section on Request Construction is part of Section 7 Addressing (as of creating this issue).  Specifically, 7.1.3: https://identity.foundation/decentralized-web-node/spec/#request-construction

However, I think it makes more sense for it to be under Section 8 where Request Objects are more properly defined.  Specifically, 8.1 seems like a more appropriate place.

**Comment by mistermoe** [comment]
agreed @shobitb, `Request Construction` feels a bit strange sitting under the `Addressing` section
<img width="400" alt="image" src="https://user-images.githubusercontent.com/4887440/177697303-18a619cc-3e21-4154-9b97-ef88d999ae5b.png">
